app/javascript/components/admin/AdminTable.js

```diff
@@ -792,6 +794,8 @@ class AdminTable extends React.Component {
             selectAll={this.state.table.selectAll}
             entryOptions={this.state.entryOptions}
             entries={this.state.query.entries}
+            orderBy={!_.isNil(this.state.query.order) ? this.state.query.order : ''}
+            sortDirection={!_.isNil(this.state.query.direction) ? this.state.query.direction : ''}
```

Comment:
Why can't these just be null as opposed to empty string? As far as I can tell it will yield the same result in the CustomTable
```suggestion
            orderBy={this.state.query.order}
            sortDirection={this.state.query.direction}
```

Reply:
good call: https://github.com/SaraAlert/SaraAlert/pull/1176/commits/f4382eda534e28d49a934c46d60e5cac3366e113

It was modified existing code that then got copied over